peer_test.go

```diff
@@ -1 +1,465 @@
 package main
+
+import (
+	"testing"
+
+	"github.com/lightningnetwork/lnd/chainntnfs"
+	"github.com/lightningnetwork/lnd/htlcswitch"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/roasbeef/btcd/btcec"
+	"github.com/roasbeef/btcd/txscript"
+	"github.com/roasbeef/btcd/wire"
+)
+
+// Test the shutdown responder's behavior if we can agree on the fee
+// immediately.
+func TestChannelClosureAcceptFeeResponder(t *testing.T) {
+	t.Parallel()
+
+	notifier := &mockNotfier{
+		confChannel: make(chan *chainntnfs.TxConfirmation),
+	}
+	broadcastTxChan := make(chan *wire.MsgTx)
+
+	responder, responderChannel, initiatorChannel, cleanUp, err :=
+		createTestPeer(notifier, broadcastTxChan)
```

Comment:
Odd line breakage here. The function should reside on the same line as the assignment operator.